docker-compose.yml

```diff
@@ -20,11 +20,13 @@ x-sentry-defaults: &sentry_defaults
     - symbolicator
     - kafka
   environment:
-    SENTRY_CONF: '/etc/sentry'
+    SENTRY_CONF: '/etc/sentry/onpremise'
     SNUBA: 'http://snuba-api:1218'
   volumes:
     - 'sentry-data:/data'
-    - './sentry:/etc/sentry'
+    - './sentry/config.yml:/etc/sentry/onpremise/config.yml'
```

Comment:
Shall we switch to https://docs.docker.com/compose/compose-file/#configs as we are changing this part?

Reply:
Sure! I assume we'd point to the config files generated after running `install.sh`, yeah?

Reply:
@BYK - looks like the `configs` feature is only supported in Docker Swarm: https://github.com/docker/compose/issues/5110

Shall we leave this as a folder mount?